about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="about.css">
+    <script src="app.js" defer></script>
+    <title>The Capstone</title>
+</head>
+<body>
+    <section class="social">
+        <ul class="social-list">
+            <li class="social-items s-logo"><a href="#" class="social-link"><img src="./images/facebook.svg" alt="facebook-logo"></a></li>
+            <li class="social-items s-logo"><a href="#" class="social-link"><img src="./images/s-twitter.svg" alt="twiteer-logo"></a></li>
+            <li class="social-items"><a href="#" class="social-link">English</a></li>
+            <li class="social-items"><a href="#" class="social-link">Mypage</a></li>
+            <li class="social-items"><a href="#" class="social-link">Logout</a></li>
+        </ul>
+    </section>
+    <header class="header">
+        <a href="#" class="logo"><img src="./images/logo.svg" alt="brand logo"></a>
+        <div class="hamburger">
```

Comment:
- [ ] Please make sure the background is transparent or the same color/pattern as the top section background.